packages/mui-material/src/Select/SelectInput.js

```diff
@@ -418,7 +418,7 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
 
   if (computeDisplay) {
     if (multiple) {
-      display = displayMultiple.reduce((prev, curr) => [prev, ', ', curr]);
+      display = displayMultiple.reduce((prev, curr) => [prev, ', ', curr], "");
```

Comment:
@Domino987 

I don't know the style guide:

`displayMultiple.reduce((prev, curr, idx) => idx === 0 ? [curr] : [prev, ', ', curr], '');`

Reply:
@mnovati I think this works :) 

@Domino987 It seems like I can't push to your branch. I think I need your permission. `remote: Permission to Domino987/material-ui.git denied to hbjORbj.` Would you make this change and as I said above add a test?

Reply:
Sorry, was afk for a few days due to work. I will add the test